### PR TITLE
enh(powershell) Add three VALID_VERBS and update the reference link (#2981)

### LINKED ISSUE #2981: (powershell) Three official-approved VALID_VERBS not been highlighted
**Describe the issue**
In the list of the [official-approved verbs](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.1), there are three of them not included in the `VALID_VERBS`:

https://github.com/highlightjs/highlight.js/blob/1e0cc762076faf49de3fe8324345085d2581f0af/src/languages/powershell.js#L27-L39

Those are:
- `Mount`
- `Build`
- `Deploy`

By the way, the provided link doesn't work anymore. Please change to a workable link, for instance: https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands

**Which language seems to have the issue?**

`powershell`

**Are you using `highlight` or `highlightAuto`?**

**Sample Code to Reproduce**

```
Mount-DiskImage -ImagePath "E:\ISO-Files\My US Visit Fall 2010 Pictures.iso"
```

**Expected behavior**

- `Mount-DiskImage` being recognized as a whole `built-in`:
```powershell
Mount-DiskImage -ImagePath "E:\ISO-Files\My US Visit Fall 2010 Pictures.iso"
```

**Additional context**

### PR DESCRIPTION
**Pull request by davidhcefx**
Resolves #2981 

### Changes

- Added three [officially-approved verbs](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.1) into `VALID_VERBS`:
  - `Mount`
  - `Build`
  - `Deploy`
- Update the reference link to: `https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands`.

### Checklist
- [x] Added markup tests, or they don't apply here because this is just a minor modification.
- [x] Updated the changelog at `CHANGES.md`
- [x] Added myself to `AUTHORS.txt`, under Contributors

### CODE REVIEW

**Comment by joshgoebel** [comment]
Will try to review on laptop tomorrow. Looked good at a glance.

**Comment by joshgoebel** [comment]
Looks good.  Please finish updating the changelog then we could merge.

**Comment by davidhcefx** [comment]
@joshgoebel I’ve complete updating `UPDATE.md`

File: CHANGES.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Language grammar improvements:
 - enh(swift) Improved highlighting for operator and precedencegroup declarations. (#2938) [Steven Van Impe][]
 - fix(xml) Support single-character namespaces. (#2957) [Jan Pilzer][]
 - enh(ruby) Support for character literals (#2950) [Vaibhav Chanana][]
+- enh(powershell) Add three VALID_VERBS and update the reference link (#2981) [davidhcefx][]
```
[the remark this answers is not in the document]
Comment (joshgoebel) [reply CHANGES.md:14]: Please also add yourself around line 40 so the `davidhcefx` link actually works.

**Review by joshgoebel — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidhcefx) [reply CHANGES.md:14]: Oh, I didn’t realize it’s a different markdown syntax. Thanks!

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
Thanks!

**Pull request merged**